eeps/eep-0057.md

```diff
@@ -200,6 +200,81 @@ alternative match expressions:
 ```
 
 
+Choice of a Delimiter for Alternative Patterns
+----------------------------------------------
+
+In this specification we propose to use the pipe `|` symbol as the
+delimiter of alternative patters for the following reasons:
+
+1. It is already used for a similar reason in type specs.
+2. It is familiar as the "or" condition to people coming from other
+   languages.
+
+Another possible alternative would be to use the `;` delimiter, which
+presently has a similar meaning as the `or` condition separating terms
+in guards, "if" statements, and list comprehensions.
```

Comment:
if statements -> if expressions

List comprehensions? Semicolons can't be used in list comprehensions.

`;` is used to separate clauses in if, case, receive, try-of-catch, fun and functions. (Maybe I forgot something.)